dagmc_stats.py

```diff
@@ -46,8 +46,11 @@ def get_entity_ranges(my_core, meshset, entity_types):
     """
 
     entity_ranges = {}
-    for entity_type in entity_types:
+    for entity_type in entity_types:#goes through each entitytype given
         entity_ranges[entity_type] = my_core.get_entities_by_type(meshset, entity_type) 
-
+        if entity_type == 11: #if the type is mbentitytypes
```

Comment:
I’d prefer to see this capability implemented in a different function:
- there is a semantic difference between the raw MOAB entities/entitySets and these tagged/categorized sets
- there is a similar semantic difference in the keys to the `entity_ranges` dictionary